renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": ["config:base", "schedule:weekends"],
+  "rebaseStalePrs": true,
+  "packageRules": [
+    {
+      "depTypeList": ["peerDependencies", "dependencies"],
+      "enabled": false
+    },
+    {
+      "packagePatterns": ["^@zendeskgarden/container"],
+      "groupName": "lerna-internal",
+      "enabled": false
+    },
+    {
+      "paths": ["packages/**"],
+      "groupName": "container-packages",
+      "minor": {
+        "groupName": "all package non-major dependencies",
+        "groupSlug": "lerna-packages-minor-patch"
+      }
+    },
+    {
+      "paths": ["package.json"],
+      "minor": {
+        "groupName": "all development non-major dependencies",
+        "groupSlug": "dev-minor-patch"
+      }
+    }
+  ],
+  "labels": ["PR: Internal :seedling:"],
+  "reviewers": ["zendeskgarden/maintainers"]
```

Comment:
Has renovate advanced to accommodate groups? (permission issue) If not, then GH user names needed here.

Reply:
Seems even user names in github don't work due to a github api bug https://renovatebot.com/docs/configuration-options/#reviewers.

I'll change this back to usernames for now